logs.go

```diff
@@ -91,6 +92,23 @@ func getLoggableResources(getPodContainers func(string, string) ([]string, error
 	return loggableResources, nil
 }
 
+// GetProjectPodsBySubstr returns a list of pod names that contain the provided string, in the named project.
+func getProjectPodNamesBySubstr(project, substr string) ([]string, error) {
+	ret := []string{}
+	pods, err := GetResourceNames(project, "pod")
```

Comment:
Although I understand the case when a constant can be useful, I do not think this should be a blanket rule that is applied to any literal string. 

Particularly in this cases like this, I do not see any advantage to having a `constant POD = "pod"` and changing this to `GetResourceNames(project, POD)`